salt/states/virtualenv_mod.py

```diff
@@ -30,6 +30,7 @@ def managed(name,
             system_site_packages=False,
             distribute=False,
             use_wheel=False,
+            no_use_wheel=False,
```

Comment:
Could you please move this down to the end of the list. Some people use positional args and this will constitute a sudden API change for them.